### ISSUE TITLE
Overflow tabs: More option tab is not reachable for keyboard users.

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4627]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.1.1 Keyboard

**Test Environment:**
- Test URL: https://clarity.design/documentation/tabs
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Open the test URL.
2. Go to the Overflow tabs.
3. Try to reach or move focus on the More tab option.

**Expected result:**
Keyboard user should be able to move focus on the More option using right arrow key.

**Actual result:**
Pressing the right arrow key expands the tab currently focused.

**Comment by chlane** [comment]
@gkhadk This is by design. Sighted keyboard-only users don't need to use the overflow button. The overflow "..." button is not a Tab. It is actually there for sighted mouse users only. Its purpose is to change the direction the Tabs are displayed in from horizontal to vertical. Sighted keyboard-only users will find that Right Arrow key will also cause this change. Blind screen reader users will not know the "..." button is there and only interact with the Tabs as expected. This is a novel variation of ARIA Tabs (https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel).

**Comment by gkhadK** [comment]
I liked the approach where keyboard navigation steps are saved, but I have encountered below 2 issues with this approach.
- Overflow menu placement is not below the More (3 dots) button, which could make keyboard user feel something is getting skipped.
- Overflow tabs can be seen quite often on mobile devices. So I tested it on iphone 11 pro with safari on iOS 13.4.1 and the behavior is - user cannot access the collapsed tab list with VoiceOver On. Verified with Linear and exploratory touch navigation. So mobile screen reader users also cannot reach those additional tabs.

**Comment by chlane** [comment]
I also verified the iOS behavior @gkhadK is talking about with my iphone 10 using Safari/VoiceOver

**Comment by walter-mair** [comment]
There is an additional related problem for 'mouse' - users, using latest Firefox/Safari Browsers:

Mouse click selection of a vertical tab, which are shown after clicking the '...' - button does not do anything
Expected behaviour: the selected tab should be selected.
Step to reproduce:
1. Open test url (with Firefox / Safari)
2. Goto the Overflow tabs.
3. Click '...'
4. Click 'Cloud' or 'Settings' => nothing happens, 'Dashboard' stays selected

Current workaround: a mix of Mouse and keyboard action is necessary!
1. Mouse click on '...' 
2. Keyboard Up-/Down- Arrow to select 'Cloud' or 'Settings'
3. Keyboard 'SPACE' to select the desired tab. => tab is selected ok.
4. Keyboard 'ESC' or Mouse click to close the vertical overflow tabs

This problem does not exist using Chrome - Browsers

**Comment by chlane** [comment]
@walter-mair Hi Walter, I can't reproduce this issue on either macOS/Firefox or macOS/Safari. Please try the Tabs overflow example at http://chrislane.info/clarity3demo1/.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
